src/openrct2/peep/PeepAnimationData.cpp

```diff
@@ -999,6 +999,15 @@ namespace OpenRCT2
                     continue;
 
                 anim.bounds = inferMaxAnimationDimensions(anim);
+
+                // Balloons, hats and umbrellas are painted in a second pass, so the inference
+                // algorithm doesn't account for those. We manually compensate for these here.
+                // Between 8-12 pixels are needed, depending on rotation, so we're generalising.
```

Comment:
This comment isn't exactly accurate, there is no second pass it when the peep is drawn and it has a balloon it attaches a child node to the parent paint node.

Reply:
Okay, let's simplify this then:
```suggestion
                // Balloons, hats and umbrellas are painted separately, so the inference
                // algorithm doesn't account for those. We manually compensate for these here.
                // Between 8-12 pixels are needed, depending on rotation, so we're generalising.
```
Is that acceptable?